packages/core/tests/stylable-processor.spec.ts

```diff
@@ -27,7 +27,20 @@ describe('Stylable postcss process', () => {
 
         expect(diagnostics.reports[0]).to.include({
             type: 'error',
-            message: 'invalid namespace'
+            message: processorWarnings.INVALID_NAMESPACE_DEF()
+        });
+    });
+
+    it('warn on empty-ish namespace', () => {
```

Comment:
What do you mean? In this test case, it'll be considered an empty namespace and will default to use the `filename` as its namespace. It will now generate a warning though.